ot/da.py

```diff
@@ -1182,6 +1183,26 @@ def inverse_transform(self, Xs=None, ys=None, Xt=None, yt=None,
 
             return transp_Xt
 
+    def normalizeCost_(self, norm):
```

Comment:
I like the normalization part but think this function should be independent from the class since it is of potential interest to any user.

I would define this function in utils.py with proper documentation. Then you can refer to this function in the doc of the classe

```python

def cost_normalization(C,norm=None):

    if norm.lower() == "median":
        C/=float(np.median(C))
    elif norm.lower() == "max":
        C/=float(np.max(C))
    elif norm.lower() == "log":
        C=np.log(1+C)
    elif norm.lower() == "loglog":
        C=np.log(1+np.log(1+C))

    return C
```